src/TestRunner.js

```diff
@@ -84,23 +91,49 @@ export default class TestRunner {
   // 3. Re-renders the app
   // 4. Runs the test
   async runTest(scope, test) {
+    const startTime = new Date();
+
     await this.component.clearAsync();
+
     if (scope.beforeEach) { await scope.beforeEach.call(scope) };
+    
     this.component.reRender();
 
     // Run the test, console logging the result.
-    let { description, f } = test;
+    let { title, label, description, f } = test;
+
+    if (!this.results.some(suite => suite.name == title)) {
+      this.results.push({
+        name: title,
+        timestamp: startTime,
+        testcases: []
+      });
+    }
+
+    let testSuite = this.results.find(suite => suite.name == title);
```

Comment:
If we're always just finding results by their name, should results be an object? I feel like there's a reason why we didn't have it as an object in the past, but this seems to indicate that's not an issue? Something about duplicate names?

Reply:
Hmmmmm :) good point, it seems as though i'd forgotten the reason it wasn't an object. With this change (and if we do start to store results as an object) if you have tests called the same thing, they will bunch together. It will grouped them together _even_ if they're in separate spec files. This does not sound ideal. Needs some thinking.

Reply:
Approach changed - no rolling up of cases into suites.